plugins/pug.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pug
+spec:
+  shortDescription: >-    
+    Checks for deprecated objects in a cluster 
+  homepage: https://github.com/rikatz/kubepug
+  caveats: |
+    * Kubepug downloads Kubernetes swagger.json to check objects and which are
+    deprecated according to its description. It uses by default "master" branch and
+    should be changed to the desired Kubernetes version with the flag --k8s-version
+    
+    * Kubepug needs permission to GET all objects in the Cluster, so you should run 
```

```suggestion
    * Kubepug needs permission to GET all objects in the Cluster
```
I think that already says enough. No need to restate it in a different way.